examples/requirements.txt

```diff
@@ -0,0 +1 @@
+accelerate # used to be installed in Amazon SageMaker environment
```

Comment:
Should we amke one folder for each example? The CV example would require more.

Reply:
for the CV example to work we would only need to add `timm`. `torchvision` should be inside the DLC since it uses the AWS PyTorch DLC as base

Reply:
Should we add `timm` to the `requirements.txt` to make both examples runnable out of the box?